cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -104,74 +104,78 @@ public static Kafka createKafkaCluster(String clusterCmNamespace, String cluster
+        Secret initialClusterCaCertSecret = createInitialCaCertSecret(clusterNamespace, clusterName,
+                AbstractModel.clusterCaSecretName(clusterName), MockCertManager.clusterCaCert());
+        Secret initialClusterCaKeySecret = createInitialCaKeySecret(clusterNamespace, clusterName,
+                AbstractModel.clusterCaKeySecretName(clusterName), MockCertManager.clusterCaKey());
         return createInitialClusterCa(clusterName, initialClusterCaCertSecret, initialClusterCaKeySecret);
     }
 
     public static ClusterCa createInitialClusterCa(String clusterName, Secret initialClusterCaCert, Secret initialClusterCaKey) {
         return new ClusterCa(new MockCertManager(), clusterName, initialClusterCaCert, initialClusterCaKey);
     }
 
-    public static ClientsCa createInitialClientsCa(String clusterCmNamespace, String clusterName) {
-        Secret initialClusterCaCertSecret = createInitialClusterCaCertSecret(clusterCmNamespace, clusterName, MockCertManager.clusterCaCert());
-        Secret initialClusterCaKeySecret = createInitialClusterCaKeySecret(clusterCmNamespace, clusterName, MockCertManager.clusterCaKey());
+    public static ClientsCa createInitialClientsCa(String clusterNamespace, String clusterName) {
+        Secret initialClusterCaCertSecret = createInitialCaCertSecret(clusterNamespace, clusterName,
+                AbstractModel.clusterCaSecretName(clusterName), MockCertManager.clusterCaCert());
+        Secret initialClusterCaKeySecret = createInitialCaKeySecret(clusterNamespace, clusterName,
+                AbstractModel.clusterCaKeySecretName(clusterName), MockCertManager.clusterCaKey());
         return createInitialClientsCa(clusterName, initialClusterCaCertSecret, initialClusterCaKeySecret);
     }
 
     public static ClientsCa createInitialClientsCa(String clusterName, Secret initialClientsCaCert, Secret initialClientsCaKey) {
         return new ClientsCa(new MockCertManager(),
-                KafkaCluster.clientsPublicKeyName(clusterName),
+                KafkaCluster.clientsCaSecretName(clusterName),
                 initialClientsCaCert,
-                KafkaCluster.clientsCASecretName(clusterName),
+                KafkaCluster.clientsCaKeySecretName(clusterName),
                 initialClientsCaKey,
                 365, 30, true);
     }
 
-    public static Secret createInitialClusterCaCertSecret(String clusterCmNamespace, String clusterName, String caCert) {
+    public static Secret createInitialCaCertSecret(String clusterNamespace, String clusterName, String secretName, String caCert) {
         return new SecretBuilder()
-        .withNewMetadata()
-            .withName(AbstractModel.getClusterCaName(clusterName))
-            .withNamespace(clusterCmNamespace)
-            .withLabels(Labels.forCluster(clusterName).withKind("Kafka").toMap())
-        .endMetadata()
-        .addToData("ca.crt", caCert)
-        .build();
+                .withNewMetadata()
```

Comment:
Argh tabs!

Reply:
I am using the default cursor position in IntelliJ when I click enter for going to the new line after `new SecretBuilder`, isn't it ok for you?